var __internal_2exceptions_8py =
[
    [ "PipError", "d6/d0a/classpip_1_1__internal_1_1exceptions_1_1_pip_error.html", null ],
    [ "DiagnosticPipError", "d8/d23/classpip_1_1__internal_1_1exceptions_1_1_diagnostic_pip_error.html", "d8/d23/classpip_1_1__internal_1_1exceptions_1_1_diagnostic_pip_error" ],
    [ "ConfigurationError", "dd/dfe/classpip_1_1__internal_1_1exceptions_1_1_configuration_error.html", null ],
    [ "InstallationError", "d9/dec/classpip_1_1__internal_1_1exceptions_1_1_installation_error.html", null ],
    [ "UninstallationError", "d3/d8e/classpip_1_1__internal_1_1exceptions_1_1_uninstallation_error.html", null ],
    [ "MissingPyProjectBuildRequires", "d4/d71/classpip_1_1__internal_1_1exceptions_1_1_missing_py_project_build_requires.html", "d4/d71/classpip_1_1__internal_1_1exceptions_1_1_missing_py_project_build_requires" ],
    [ "InvalidPyProjectBuildRequires", "de/dc9/classpip_1_1__internal_1_1exceptions_1_1_invalid_py_project_build_requires.html", "de/dc9/classpip_1_1__internal_1_1exceptions_1_1_invalid_py_project_build_requires" ],
    [ "NoneMetadataError", "de/d1a/classpip_1_1__internal_1_1exceptions_1_1_none_metadata_error.html", "de/d1a/classpip_1_1__internal_1_1exceptions_1_1_none_metadata_error" ],
    [ "UserInstallationInvalid", "d8/d57/classpip_1_1__internal_1_1exceptions_1_1_user_installation_invalid.html", "d8/d57/classpip_1_1__internal_1_1exceptions_1_1_user_installation_invalid" ],
    [ "InvalidSchemeCombination", "df/d17/classpip_1_1__internal_1_1exceptions_1_1_invalid_scheme_combination.html", "df/d17/classpip_1_1__internal_1_1exceptions_1_1_invalid_scheme_combination" ],
    [ "DistributionNotFound", "d1/d76/classpip_1_1__internal_1_1exceptions_1_1_distribution_not_found.html", null ],
    [ "RequirementsFileParseError", "d6/d2e/classpip_1_1__internal_1_1exceptions_1_1_requirements_file_parse_error.html", null ],
    [ "BestVersionAlreadyInstalled", "d4/d60/classpip_1_1__internal_1_1exceptions_1_1_best_version_already_installed.html", null ],
    [ "BadCommand", "db/d48/classpip_1_1__internal_1_1exceptions_1_1_bad_command.html", null ],
    [ "CommandError", "db/ddf/classpip_1_1__internal_1_1exceptions_1_1_command_error.html", null ],
    [ "PreviousBuildDirError", "d1/d0f/classpip_1_1__internal_1_1exceptions_1_1_previous_build_dir_error.html", null ],
    [ "NetworkConnectionError", "db/dd8/classpip_1_1__internal_1_1exceptions_1_1_network_connection_error.html", "db/dd8/classpip_1_1__internal_1_1exceptions_1_1_network_connection_error" ],
    [ "InvalidWheelFilename", "dd/d85/classpip_1_1__internal_1_1exceptions_1_1_invalid_wheel_filename.html", null ],
    [ "UnsupportedWheel", "d2/d85/classpip_1_1__internal_1_1exceptions_1_1_unsupported_wheel.html", null ],
    [ "InvalidWheel", "d3/dd5/classpip_1_1__internal_1_1exceptions_1_1_invalid_wheel.html", "d3/dd5/classpip_1_1__internal_1_1exceptions_1_1_invalid_wheel" ],
    [ "MetadataInconsistent", "dd/d0c/classpip_1_1__internal_1_1exceptions_1_1_metadata_inconsistent.html", "dd/d0c/classpip_1_1__internal_1_1exceptions_1_1_metadata_inconsistent" ],
    [ "InstallationSubprocessError", "d2/d88/classpip_1_1__internal_1_1exceptions_1_1_installation_subprocess_error.html", "d2/d88/classpip_1_1__internal_1_1exceptions_1_1_installation_subprocess_error" ],
    [ "MetadataGenerationFailed", "d6/dc0/classpip_1_1__internal_1_1exceptions_1_1_metadata_generation_failed.html", "d6/dc0/classpip_1_1__internal_1_1exceptions_1_1_metadata_generation_failed" ],
    [ "HashErrors", "da/d1a/classpip_1_1__internal_1_1exceptions_1_1_hash_errors.html", "da/d1a/classpip_1_1__internal_1_1exceptions_1_1_hash_errors" ],
    [ "HashError", "d5/d33/classpip_1_1__internal_1_1exceptions_1_1_hash_error.html", "d5/d33/classpip_1_1__internal_1_1exceptions_1_1_hash_error" ],
    [ "VcsHashUnsupported", "dd/df6/classpip_1_1__internal_1_1exceptions_1_1_vcs_hash_unsupported.html", null ],
    [ "DirectoryUrlHashUnsupported", "dc/d66/classpip_1_1__internal_1_1exceptions_1_1_directory_url_hash_unsupported.html", null ],
    [ "HashMissing", "d4/dcb/classpip_1_1__internal_1_1exceptions_1_1_hash_missing.html", "d4/dcb/classpip_1_1__internal_1_1exceptions_1_1_hash_missing" ],
    [ "HashUnpinned", "da/d70/classpip_1_1__internal_1_1exceptions_1_1_hash_unpinned.html", null ],
    [ "HashMismatch", "da/dc1/classpip_1_1__internal_1_1exceptions_1_1_hash_mismatch.html", "da/dc1/classpip_1_1__internal_1_1exceptions_1_1_hash_mismatch" ],
    [ "UnsupportedPythonVersion", "d0/db6/classpip_1_1__internal_1_1exceptions_1_1_unsupported_python_version.html", null ],
    [ "ConfigurationFileCouldNotBeLoaded", "d3/d00/classpip_1_1__internal_1_1exceptions_1_1_configuration_file_could_not_be_loaded.html", "d3/d00/classpip_1_1__internal_1_1exceptions_1_1_configuration_file_could_not_be_loaded" ],
    [ "ExternallyManagedEnvironment", "d6/d6d/classpip_1_1__internal_1_1exceptions_1_1_externally_managed_environment.html", "d6/d6d/classpip_1_1__internal_1_1exceptions_1_1_externally_managed_environment" ],
    [ "pip._internal.exceptions._is_kebab_case", "d4/dc3/namespacepip_1_1__internal_1_1exceptions.html#a9746fdd985a67ad9341d74a2a3234f00", null ],
    [ "pip._internal.exceptions._prefix_with_indent", "d4/dc3/namespacepip_1_1__internal_1_1exceptions.html#a5cc29aebb3e7c201ac3113d65c689b5d", null ],
    [ "pip._internal.exceptions._DEFAULT_EXTERNALLY_MANAGED_ERROR", "d4/dc3/namespacepip_1_1__internal_1_1exceptions.html#a26038942f3347b376077e51afeee8d20", null ],
    [ "pip._internal.exceptions.logger", "d4/dc3/namespacepip_1_1__internal_1_1exceptions.html#a0b14e488ae28d98d262453f3e9cd6e4d", null ]
];